client-datastore-cassandra/src/main/java/com/telecominfraproject/wlan/client/datastore/cassandra/ClientDAO.java

```diff
@@ -142,7 +142,11 @@
     private static final String CQL_DELETE_BLOCKED_LIST = 
     		"delete from client_blocklist where customerId = ? and  macAddress = ?";
     
-    private static final String CQL_GET_BY_MAC_STRING = 
+    // Statements using client_by_mac_string table
+    private static final String CQL_GET_CLIENT_MAC_BY_CUSTOMER_ID = 
+    		"select customerId, macAddress from client_by_mac_string where customerId = ? and macAddressString like ?";
```

Comment:
nope. you probably meant just 
"select customerId, macAddress from client_by_mac_string where customerId = ?";